src/HolographicCamera.Unity/Assets/HolographicCamera/Scripts/KeepDeviceActive.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+#if UNITY_WSA && WINDOWS_UWP
+    using System;
+    using Windows.System.Display;
+#endif
+
+public class KeepDeviceActive : MonoBehaviour
+{
+#if UNITY_WSA && WINDOWS_UWP
+    private static DisplayRequest displayRequest;
+#endif
+
+    private void Start()
+    {
+        CreateDisplayRequest();
+    }
+
+    private void OnDestroy()
+    {
+#if UNITY_WSA && WINDOWS_UWP
+        if (displayRequest != null)
+        {
+            try
+            {
+                displayRequest.RequestRelease();
+                Debug.Log("Display request released.");
+            }
+            catch (Exception e)
+            {
+                Debug.LogError($"Error releasing display request: {e.Message}");
+            }
+        }
+#endif
+    }
+
+    private void CreateDisplayRequest()
+    {
+#if UNITY_WSA && WINDOWS_UWP
+        UnityEngine.WSA.Application.InvokeOnUIThread(() =>
+        {
+            if (displayRequest == null)
+            {
+                try
+                {
+                    displayRequest = new DisplayRequest();
+                }
+                catch (Exception e)
+                {
+                    Debug.LogError($"Error creating display request: {e.Message}");
+                }
```

Comment:
Is it really possible for the construction of the DisplayRequest to throw an exception?

Reply:
I'm not sure why it would, but this mimics the sample behavior where it suggests that it can:
https://docs.microsoft.com/en-us/uwp/api/windows.system.display.displayrequest?view=winrt-18362